sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -212,6 +212,7 @@ def _delete_path_options(**kwargs):
         options = {
             'lease_access_conditions': access_conditions,
             'modified_access_conditions': mod_conditions,
+            'cls': return_response_headers,
```

Comment:
if we add line 215 probably line 221 should also be changed since it has return type now?